spec/models.spec/users.spec.js

```diff
@@ -37,10 +37,42 @@ describe('UserSchema', function() {
         expect(info.name).toBe('Kingsley');
         expect(info.emailAddress).toBe('chitech247@gmial.com');
         });
-    });*/
+    });
      it('should throw an error when data is not sufficient, name or emailAddress', function() {
         User.create({name: 'Kingsley'}, function(err, info){
         expect(err).toBeDefined();
         });
     });
+   it('should find users', function() {
+        User.find(function(err, info){
+            if(err){
```

Comment:
Thanks


On Thu, May 7, 2015 at 8:23 AM, Olatunbosun Olanrewaju <
notifications@github.com> wrote:

> In spec/models.spec/users.spec.js
> <https://github.com/andela-kochu/freelance/pull/1#discussion_r29829572>:
>
> >       it('should throw an error when data is not sufficient, name or emailAddress', function() {
> >          User.create({name: 'Kingsley'}, function(err, info){
> >          expect(err).toBeDefined();
> >          });
> >      });
> > +   it('should find users', function() {
> > +        User.find(function(err, info){
> > +            if(err){
>
> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andela-kochu/freelance/pull/1/files#r29829572>.
>



-- 

*--*
*See what WIRED <http://www.wired.com/2014/12/andela/> and CNN
<http://www.cnn.com/2014/12/24/tech/africa-tech-startups-2014/> are saying
about Andela!*

*Kingsley Ochu | Software Developer | Andela*
(c) 234 (0) 80 649 49432 <08064949432> | Andela.co <http://andela.co/> |
@kingsleyochu <http://twitter.com/kingsleyochu>